doc/apiv3-documentation.apib

```diff
@@ -2139,6 +2139,84 @@ For more details and all possible responses see the general specification of [Fo
                 "message": "The request body was not empty."
             }
 
+
+# Group Revisions
+
+Revisions are sets of updates to files in the context of repositories linked in OpenProject.
+
+## Linked Properties
+|  Link            | Description                                         | Type          | Constraints | Supported operations |
+|:----------------:| --------------------------------------------------- | ------------- | ----------- | -------------------- |
+| self             | This revision                                       | Revision      | not null    | READ                 |
+| project          | The project to which the revision belongs           | Project       | not null    | READ                 |
```

Comment:
In the long term this should rather be a repository, shouldn't it?

Or do you think that we can entirely leave out the concept of repositores from the APIv3 and replace it by projects? (serious question)

I am not quite sure which things a repository adds to a project (aside from revisions ^^).

My short-term question would be: Do we need a link to the project now, if we know that we will later *replace* it with a different link?